build.gradle

```diff
@@ -6,7 +6,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.5.0'
+        classpath 'com.android.tools.build:gradle:3.6.3'
```

Comment:
can we do 4.0 instead? AS 4.0 stable was released last week

Reply:
4.0.0 introduced some breaking change, which need to be resolved. I can try to resolve the issues.

Reply:
It seems there is an incompatibility issue with the `android-sdk-versions-plugin` and gradle plugin 4.0.0, the error is printed as follows:
```
Cannot cast object 'task ':MapboxGLAndroidSDK:generateDebugBuildConfig' property 'versionCode'' with class 'org.gradle.api.internal.provider.DefaultProperty' to class 'java.lang.Integer'
```
cc @harvsu